### DISCUSSION TITLE
Requesting Reports from Local Instance

### DISCUSSION THREAD

**Discussion by hfreeman02** [biigle/core#854] [Q&A] [answered]
Hello,

We now have Biigle downloaded as a local instance and are accessing videos from a remote location. However, it does not seem to be working for downloading reports. We have submitted requests for reports around 2-3 days ago and have yet to receive an email. When we were doing this on Biigle online (biigle.de) this took at most 5 minutes! Is this delay expected or is there a better way to approach this?

Thanks!

**Comment by mzur** [comment, marked as answer]
Do you find the reports for download when you go to the search page and select the "reports" tab? If the reports are there and you can download them, it seems like your local instance is unable to send emails. Please check the [email credentials](https://github.com/biigle/biigle/blob/2e88a538528bd62508083c8fe013994cee4d1a1f/build/.env.example#L58-L64) in the build configuration.

If you have no way of sending emails with your local instance, there is also another way of using in-app notifications. This is a little more complicated so I will only explain this in case you need it :grin:

**Reply by danieblue02** [reply]
Thank you!
